resources/normal/base/resource_map.json

```diff
@@ -3080,6 +3290,132 @@
         "small": "RESOURCE_ID_BLE_HRM_SHARING_SMALL",
         "large": "RESOURCE_ID_BLE_HRM_SHARING_LARGE"
       }
+    },
+    {
+      "name": "NOTIFICATION_BEEPER",
+      "id": 120,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_BEEPER_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_BEEPER_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_BEEPER_LARGE"
+      }
+    },
+    {
+      "name": "NOTIFICATION_DISCORD",
+      "id": 121,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_DISCORD_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_DISCORD_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_DISCORD_LARGE"
+      }
+    },
+    {
+      "name": "NOTIFICATION_BLUESKY",
+      "id": 122,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_BLUESKY_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_BLUESKY_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_BLUESKY_LARGE"
+      }
+    },
+    {
+      "name": "NOTIFICATION_DUOLINGO",
+      "id": 123,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_DUOLINGO_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_DUOLINGO_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_DUOLINGO_LARGE"
+      }
+    },
+    {
+      "name": "NOTIFICATION_ELEMENT",
+      "id": 124,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_ELEMENT_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_ELEMENT_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_ELEMENT_LARGE"
+      }
+    },
+    {
+      "name": "NOTIFICATION_GOOGLE_CHAT",
+      "id": 125,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_GOOGLE_CHAT_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_GOOGLE_CHAT_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_GOOGLE_CHAT_LARGE"
+      }
+    },
+    {
+      "name": "NOTIFICATION_GOOGLE_TASKS",
+      "id": 126,
+      "sizes": {
+        "tiny": "RESOURCE_ID_NOTIFICATION_GOOGLE_TASKS_TINY",
+        "small": "RESOURCE_ID_NOTIFICATION_GOOGLE_TASKS_SMALL",
+        "large": "RESOURCE_ID_NOTIFICATION_GOOGLE_TASKS_LARGE"
```

Comment:
This seems like not the worst thing to replace google inbox with, unless you really do not want to mess with replacing old ids